src/models/model_base.py

```diff
@@ -1,10 +1,30 @@
+from sqlalchemy.ext.declarative import declared_attr
 from sqlalchemy.orm import validates
+
 from src import db
+from src.event_dispatcher import EventDispatcher
+from src.services.event_service_base import Event, EventType
 
 
 class ModelBase(db.Model):
     __abstract__ = True
 
+    @declared_attr
+    def created_on(cls):
+        return db.Column(db.DateTime, server_default=db.func.now())
+
+    @declared_attr
+    def updated_on(cls):
+        return db.Column(db.DateTime, server_default=db.func.now(), onupdate=db.func.now())
+
+    @classmethod
+    def find_by_uuid(cls, device_uuid):
+        return cls.query.filter_by(uuid=device_uuid).first()
```

Comment:
Nice one, instead of having these on all models it's better to introduce here.

Remove those two fields and a function on `model_wires_plat.py > wires_plat` also, coz it's already introduced here.